README.md

```diff
@@ -34,6 +34,8 @@ targets: [
 
 Next, copy/paste the `Resources/Views/Submissions` folder into your project in order to be able to use the provided Leaf tags. These files can be changed as explained in the [Leaf Tags](#leaf-tags) section, however it's recommended to copy this folder to your project anyway. This makes it easier for you to keep track of updates and your project will work if you decide later on to not use your own customized leaf files.
 
+> Right now the provided Leaf templates are dependening on the [Bootstrap package](https://github.com/nodes-vapor/bootstrap). This will change in the future, but until then, the consumer is forced to either include this package in their project or overwrite the paths to some personal leaf files.
```

Comment:
dependening - depending
last sentence: "This will change in the future. For the moment, the consumer of this package will need to include `Bootstrap` or adapt the provided leaf files or replace them entirely.